inc/class-upgrade.php

```diff
@@ -120,6 +124,37 @@ private function upgrade_22() {
 	}
 
 	/**
+	 * Performs upgrade function to WP SEO 2.3
+	 */
+	private function upgrade_23() {
+		$wp_query = new WP_Query( 'meta_key=_yoast_wpseo_sitemap-include&meta_value=never&order=ASC' );
```

Comment:
If you don't specify a `post_type`, WordPress will query only `post`s, so we should add `post_type=any` to migrate all exclusions.